pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -35,12 +35,13 @@ type Alertmanager interface {
 	SaveAndApplyConfig(ctx context.Context, config *apimodels.PostableUserConfig) error
 	SaveAndApplyDefaultConfig(ctx context.Context) error
 	GetStatus() apimodels.GettableStatus
```

Comment:
Probably add ctx to all methods that do not have it now? Not in this PR though

Reply:
Yes, I'll do that for other methods too, I wanted to keep the diff as small as possible.